cfme/common/vm.py

```diff
@@ -325,11 +325,8 @@ def find_quadicon(self, from_any_provider=False, use_search=True):
         if 'Grid View' != view.toolbar.view_selector.selected:
             view.toolbar.view_selector.select('Grid View')
 
-        if use_search:
-            search.normal_search(self.name)
-
         try:
-            return view.entities.get_entity(name=self.name, surf_pages=True)
+            return view.entities.get_entity(name=self.name, surf_pages=True, use_search=True)
```

Comment:
**Required/Question**

Why pass `True` instead of `use_search` that was passed into find_quadicon?

Reply:
it doesn't have anything behind except sleeplessness :). fixed